xapian-core/queryparser/termgenerator_internal.cc

```diff
@@ -241,15 +241,15 @@ TermGenerator::Internal::index_text(Utf8Iterator itor, termcount wdf_inc,
 {
     bool cjk_ngram = (flags & FLAG_CJK_NGRAM) || CJK::is_cjk_enabled();
 
-    int stop_mode = STOPWORDS_INDEX_UNSTEMMED_ONLY;
+    //int stop_mode = STOPWORDS_INDEX_UNSTEMMED_ONLY;
 
-    if (!stopper) stop_mode = STOPWORDS_NONE;
+    if (!stopper) stop_mode = TermGenerator::STOPWORDS_NONE;
```

Comment:
This will change the stored `stop_mode`, which I don't think is the right thing to do. (Otherwise, if I set the stopping strategy but don't set the stopper, call `index_text()` and then set the stopper, it won't do quite what I expect.)